src/mscorlib/shared/System/IO/Path.Unix.cs

```diff
@@ -189,6 +194,14 @@ public static string GetPathRoot(string path)
             return IsPathRooted(path) ? PathInternal.DirectorySeparatorCharAsString : String.Empty;
         }
 
+        public static ReadOnlySpan<char> GetPathRoot(ReadOnlySpan<char> path)
+        {
+            if (path.IsEmpty)
```

Comment:
This whole check is unnecessary. IsPathRooted below will return false for empty path that takes care of it.